package.json

```diff
@@ -8,8 +8,11 @@
   },
   "dependencies": {
     "cors": "^2.8.5",
+    "cron": "^1.7.1",
```

Comment:
I'm actually not sure why do you need all this stuff https://github.com/kelektiv/node-cron/blob/master/lib/cron.js if there is just setInterval :)